### PR TITLE
Improvements to recording dialog

### PR DESCRIPTION
**Pull request by microbit-robert**
- Use a standard dialog
- Always show instructions, countdown and progress bar
- Title to inform the user which action is being recorded
- General size, layout, colour changes

Draft as requires internal discussion before merging. See https://microbit-global.monday.com/boards/1356069004/pulses/1356409062.

### CODE REVIEW

File: src/components/dialogs/StandardDialog.svelte [hunk 1]
```diff
@@ -21,6 +21,8 @@
   // of using svelte-headlessui dialogs
   $: if (isOpen) {
     dialog.open();
+  } else {
+    dialog.close();
```
Comment (microbit-robert) [inline src/components/dialogs/StandardDialog.svelte:25]: Needed so the dialog closes automatically when the open prop is no longer true.

File: src/components/dialogs/StandardDialog.svelte [hunk 2]
```diff
@@ -29,7 +31,7 @@
   });
 </script>
 
-<div class="relative z-10">
+<div class="fixed z-10">
```
Comment (microbit-robert) [inline src/components/dialogs/StandardDialog.svelte:34]: Keeps dialogs out of DOM flow.

**Comment by microbit-robert** [comment]
https://recording-dialog.ml-trainer.pages.dev/

File: src/messages/ui.en.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
   "actions.close": "Close",
 
-  "alert.data.classNameLengthAlert": "Names must consists of maximum {maxLen} characters.",
```
Comment (microbit-matt-hillsdon) [inline src/messages/ui.en.json:8]: Spotted odd phrasign when checking dialog title length

**Review by microbit-grace — APPROVED** [review]
LGTM

**Pull request merged**